quote_load.py

```diff
@@ -0,0 +1,40 @@
+import requests
+import logging
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger("Quote_Load_Logger")
+
+
+class QuoteException(Exception):
+    pass
+
+
+class NoResponseFromQuoteUrl(QuoteException):
+    pass
+
+
+def get_quote(quote_url):
+    try:
+        logger.info('Looking for the inspiring quote...')
+        quote_resp = requests.get(quote_url)
+        quote_text = quote_resp.json()[0]['q']

```

Comment:
2 .json() statements could be quite heavy. Try to move them to a separate statement